### DISCUSSION TITLE
Service continuation after unsubscription

### DISCUSSION THREAD

**Discussion by thymaro** [simple-login/app#1649] [Q&A] [answered]
Before I subscribe to a paid account, I need to know something. If I ever unsubscribe, what happens to my existing aliases? As I would use the aliases to create accounts on other websites, what would I need to do if I ever choose to downgrade my SL account to either use another aliasing service from then on or just use a personal email account?

Would I need to change my login e-mail on those other websites or would I still get the forwards from SL? How long would I have the time to change all my logins before SL stops forwarding emails? Is the problem clear enough?

I have a done a search for similar discussions, but the keywords I used for my search just provided a bunch of stuff for unsubscribing from services on other websites. Finding out what happens when trying to unsubscribe from SL is not straitforward 😄

**Comment by ghost** [comment, marked as answer]
You can see the answer to your question [here](https://simplelogin.io/pricing/) 

For your convenience I’m uploading a screenshot:
![B62F71E5-099A-490C-AEB9-7CF5B816788A](https://user-images.githubusercontent.com/128491613/226630424-fa1e3595-49d8-4af0-989d-7a5b8ec12c33.jpeg)

**Reply by thymaro** [reply]
gotcha! thanks!
